tests/backup/backup_helper.go

```diff
@@ -2026,3 +2028,30 @@ func MapToKeyValueString(m map[string]string) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// LicenseConsumedCountVerify verifies the consumed license count for px-backup
+func LicenseConsumedCountVerify(ctx context.Context, OrgId string, expectedLicenseConsumedCount int64) error {
```

Comment:
Rename this to `VerifyLicenseConsumedCount`

Reply:
done